treelite.sh

```diff
@@ -37,8 +37,8 @@ module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@
 module load BASE/1.0
 # Our environment
 set osname [uname sysname]
-setenv TREELITE_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
-prepend-path PATH \$::env(TREELITE_ROOT)/bin
-prepend-path LD_LIBRARY_PATH \$::env(TREELITE_ROOT)/lib
-$([[ ${ARCHITECTURE:0:3} == osx ]] && echo "prepend-path DYLD_LIBRARY_PATH \$::env(TREELITE_ROOT)/lib")
+prepend-path PATH \$::env(BASEDIR)/$PKGNAME/\$version/bin
+prepend-path ROOT_INCLUDE_PATH \$::env(BASEDIR)/$PKGNAME/\$version/include
+prepend-path LD_LIBRARY_PATH \$::env(BASEDIR)/$PKGNAME/\$version/lib
+$([[ ${ARCHITECTURE:0:3} == osx ]] && echo "prepend-path DYLD_LIBRARY_PATH \$::env(BASEDIR)/$PKGNAME/\$version/lib")
```

Comment:
While you are at it, can you drop `DYLD_LIBRARY_PATH`? I think we do not need it anymore with the new way relocation is done.

Reply:
OK! I push it here, but maybe could you wait for merging that I test locally in my mac? I feel a bit uneasy to do this change with the macos tests down.